
project device/phh/treble/
diff --git a/rw-system.sh b/rw-system.sh
index 6867332..46fb44a 100644
--- a/rw-system.sh
+++ b/rw-system.sh
@@ -843,9 +843,10 @@ if [ -f /system/phh/secure ] || [ -f /metadata/phh/secure ];then
     copyprop ro.product.manufacturer ro.vendor.product.manufacturer
     copyprop ro.system.product.manufacturer ro.product.vendor.manufacturer
     copyprop ro.product.manufacturer ro.product.vendor.manufacturer
-    (getprop ro.vendor.build.security_patch; getprop ro.keymaster.xxx.security_patch) |sort |tail -n 1 |while read v;do
-        [ -n "$v" ] && resetprop_phh ro.build.version.security_patch "$v"
-    done
+    # Disabled: Don't override system SPL with vendor SPL (Settings > About Device fix)
+    #(getprop ro.vendor.build.security_patch; getprop ro.keymaster.xxx.security_patch) |sort |tail -n 1 |while read v;do
+    #    [ -n "$v" ] && resetprop_phh ro.build.version.security_patch "$v"
+    #done
 
     resetprop_phh ro.build.user nobody
     resetprop_phh ro.build.host android-build
